modules/main-rules/pages/assault-moves-and-charges.adoc

```diff
@@ -26,4 +26,10 @@ This is the only time that you may move your units into base contact with enemy
 * You don't have to declare and make a charge into close combat unless you want to, with one exception: If you can only meet the 'must move towards the enemy' requirement by moving toward an enemy that has units that are within 5cm of your units, then you must charge and place units in base contact.
 * For a charge, you may double the Speed value of *infantry* units *if* this will bring them into base contact with enemy units.
 * If you don't get at least one unit into base contact with an enemy then there is no charge and there will be no close combat.
-(Though there will be a firefight if any units are within 15cm of the enemy.)
\ No newline at end of file
+(Though there will be a firefight if any units are within 15cm of the enemy.)
+* Where a unit has a split Speed value such as 20/30 cm, you may use the second value if this would bring the unit into contact with an enemy. You *cannot* double the Speed value of the unit during an Assault move. +
```

Comment:
Rewrite of the split Speed text to fit better.

```suggestion
* For units with a split Speed value such as 20/30 cm, you may use the second value for a charge *if* this will bring them into base contact with enemy units. This is *instead* of doubling the units' Speed value. +
```

Reply:
Will resolve conversation, as outdate - newer commit has fixed this.